src/asio_service.cxx

```diff
@@ -735,8 +738,9 @@ class rpc_session
        try {
         ptr<buffer> resp_ctx = resp->get_ctx();
         int32 resp_ctx_size = (resp_ctx) ? resp_ctx->size() : 0;
+        int32 result_code_size = sizeof(int32_t);
```

Comment:
Instead of this, can you please use
```c++
int32 result_code_size = sizeof(cmd_result_code);
```

Reply:
ok
my concern was that we will use the put_i32 and get_i32 functions, so it result_code_size is not sizeof(int32_t), weird things may happen

Reply:
Yeah, good point, that makes more sense. Let me make a change to use `sizeof(int32_t)` and instead put `int32_t` to `cmd_result_code` definition.